packages/components/src/components/tabs/_tabs.scss

```diff
@@ -407,6 +407,28 @@
     padding: $carbon--spacing-05;
   }
 
+  .#{$prefix}--tabs--container .#{$prefix}--tab-content .#{$prefix}--text-input,
```

Comment:
It seems like these are adjacent selectors, so it's not picking up the styles. 

<img width="814" alt="Screen Shot 2020-03-13 at 10 54 17 AM" src="https://user-images.githubusercontent.com/11928039/76646909-10488f80-6519-11ea-9af4-f601c05d07e9.png">



Seems like it is fixed if we do something like 

```scss
.#{$prefix}--tabs--container ~ .#{$prefix}--tab-content .#{$prefix}--text-input
```

etc for each of them.

Reply:
🤦‍♂ I must've been using a cached build or something. Fixed and I added an input to one of the stories for easier testing.